src/Middleware/RateLimiting/src/EnableRateLimitingAttribute.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.RateLimiting;
+
+/// <summary>
+/// Metadata that provides endpoint-specific request rate limiting.
```

Comment:
"Replaces any policies currently applied to the endpoint, still runs the global limiter"
I think we should try to be more descriptive about what exactly APIs do.